rwengine/src/objects/CharacterObject.cpp

```diff
@@ -198,11 +198,8 @@ glm::vec3 CharacterObject::updateMovementAnimation(float dt) {
         }
     }
 
-    if (controller) {
-        if (static_cast<PlayerController*>(controller)->isAdrenalineActive() &&
-            movementAnimation == animations->animation(AnimCycle::WalkStart)) {
-            animationSpeed *= 2;
-        }
+    if (isPlayer() && (movementAnimation == animations->animation(AnimCycle::WalkStart))) {
```

Comment:
Now you have those superfluous parentheses again... :D

Reply:
![image](https://user-images.githubusercontent.com/4138939/41470478-fbeb45ae-70b0-11e8-9d40-225d91c17902.png)